File: DSA/Data_Structure/Matrix/Easy/Richest_Customer_Wealth/Solutions/solve.py
def maximumWealth(accounts):
    res = 0
    for row in accounts:
        total = 0
        for num in row:
            total += num
        if(res < total):
            res = total
    return res

def main():
    print(maximumWealth([[1,3,4],[2,5,6]]))

main()      

File: DSA/Data_Structure/Matrix/Easy/Richest_Customer_Wealth/Solutions/solve.cpp
#include <iostream>
#include <climits>
using namespace std;

class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int res = 0;
        for(int i = 0; i < accounts.size(); i++){
            int total = 0;
            for(int j = 0; j < accounts[i].size(); j++){
                total += accounts[i][j];
            }
            res = std::max(res, total);
        }
        return res;
    }
};

int main()
{
    int a[R][C] = { { 1, 2, 3, 4 },
                    { 5, 6, 7, 8 },
                    { 9, 10, 11, 12 },
                    { 13, 14, 15, 16 } };
 
    // Function Call
    maximumWealth(a);
    return 0;
}
